critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
```

Comment:
If it's possible to just make it a dependency and modify defaults externally, I'd also prefer to do that.